server/test/unit/controllers/mailing.spec.js

```diff
@@ -299,9 +302,10 @@ describe('mailing controllers', () => {
             eventContact: 'hello@coolestprojects.org',
             eventUrl: `${process.env.HOSTNAME}/events/intl`,
             requiresApproval: false,
+            emailIteration: 1,
             intl: true,
           },
-          categories: ['coolest-projects', 'cp-intl-confirm-attendance'],
+          categories: ['coolest-projects', 'cp-intl-1-confirm-attendance'],
```

Comment:
Hoe come the category rename?

Reply:
It takes into account the iteration number so that we can identify what impact has `resend the confirmation email with more pressing text each time` and separate each iteration's result
Example: 
 - First iteration could have a 70% open rate, 10% click through 
 - 2nd iteration could have a 40% open rate, 50% click through
 - 3rd iteration could have a 10% open rate, 10% click through